oidc-signature-extension.md

```diff
@@ -262,200 +208,224 @@ The following is a non-normative example of the claims in a Request Object befor
-
-<a name="signature-scopes"></a>
-### 3.3. Signature Scopes
-
-This section defines two scopes that represent the signature models described in [chapter 2](#signature-models). The scopes are used when a client sends an authentication request that is a request for signature. 
+<a name="security-requirements"></a>
+#### 3.1.2. Security Requirements
 
-The scopes defined below only declare signature specific claims. A client requesting signature typically also includes other scopes/claims in the authentication request (for release of user identity information, and possibly also for extended authentication information). 
+The contents of the `https://id.oidc.se/param/signRequest` parameter hold the data to be signed, and it is essential
+that no party can alter this while the request message is in transit. Therefore, the following security requirements
+apply for Relying Parties and OpenID Providers that are compliant with this specification:
 
-<a name="delegated-signing"></a>
-#### 3.3.1. Delegated Signing
+* If the signature request parameter is included as a custom request parameter (see [3.1.1.1](#as-a-custom-request-parameter) 
+above), its value represented as a JWT MUST be signed by the client's registered key, and MAY be encrypted to the recipient's
+registered public key.
 
-**Scope:** `https://scopes.oidc.se/1.0/delegatedSign`
+* If the signature request parameter is part of a Request Object according to section [3.1.1.2](#placed-in-a-request-object)
+above, the entire Request Object JWT MUST be signed by the client's registered key, and MAY be encrypted to the recipient's
+registered public key. 
 
-**Description:** This scope services two purposes; it indicates for the OpenID Provider that the current authentication request is a request for delegated signing, and, it requests the claims declared below.
+<a name="signature-scope"></a>
+### 3.2. Signature Scope
 
-| Claim | Description/comment | Requirement |
-| :--- | :--- | :--- |
-| `https://claims.oidc.se/`<br />`1.0/userSignature` | The resulting user signature. | Mandatory |
+**Scope:** `https://id.oidc.se/scope/sign`
 
-<a name="federated-signing"></a>
-#### 3.3.2. Federated Signing
+**Description:** The scope has two purposes; it indicates for the OpenID Provider that the request in which the scope is
+included is a "signature request", and it requests the claims declared in the table below.
 
-**Scope:** `https://scopes.oidc.se/1.0/federatedSign`
+| Claim | Description/comment | Reference | Requirement |
+| :--- | :--- | :--- | :--- |
+| `https://id.oidc.se/`<br />`claim/userSignature` | The signature that is the result of the user signing process at the OP. | \[[OIDC.Sweden.Attr](#attr-spec)\] | REQUIRED |
+| `auth_time` | The time where the signature was created. | \[[OpenID.Core](#openid-core)\] | REQUIRED |
```

Comment:
Nitpick: I believe the wording should use "The time **when** the ..... " rather than "The time **where** the ....".